senderworker/src/main/resources/logback.xml

```diff
@@ -0,0 +1,26 @@
+<configuration>
```

Comment:
My understanding is that this logback configuration allows for structured json logging. Any existing logs we use to populate dashboards in kibana will continue to work, albeit they should be logged under a different `appName`.

In the sender lambdas though we emit metrics using the embedded log structure (logs that have a key `_aws`). I _think_ the logging configuration for ec2s means the logs are not parsed by cloudwatch so we will not emit the same metrics when the ec2 service is running.

Perhaps not a consideration for now, but if we're not parsing/emitting metrics via cloudwatch we may want to consider setting up a grafana dashboard that could consume/collate metrics from different sources, OR we could change the existing lambda functions to just use the PUT metric api instead of the structured logs...

Reply:
I agree, it can be a follow-up task :)

Reply:
I have created a JIRA ticket [LIVE-4746](https://theguardian.atlassian.net/browse/LIVE-4746) for this follow-up task.